argo-cd-apps/base/member/infra-deployments/konflux-public/konflux-public.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: konflux-public
```

Comment:
Disclaimer, this is a subjective comment. "Konflux-public" name might be confused with internal vs public instances. I would have named it something more like "konflux-info"

Feel free to ignore this comment :)

Reply:
I changed it to `konflux-info`